src/notebook/index.css

```diff
@@ -9,6 +9,11 @@
 }
 
 
+.p-Widget * + p { 
```

Comment:
We were running into an issue where there wouldn't be any spacing after a heading when there was a cell that had multiple p and h elements within it. I was thinking of some kind of workaround involving class names but I didn't want to go that deep into this issue.

The only reason we did it this way was that when we inspected the Notebook in chrome to try and match the style, we saw pretty much that exact line of code, tried it out, and it worked. If it is too expensive to use then we can take it out, it really doesn't make a difference. 

I could make a rule like `h1 + p { /* rule stuff */ }` and apply that for each header, but that would be ugly and I don't know enough about CSS to do something that nasty.